src/Notifications/Wars/Discord/AllyJoinedWarDefenderMsg.php

```diff
@@ -63,7 +63,7 @@ public function populateMessage(DiscordMessage $message, $notifiable)
         $message
             ->content('A new member has been enroll in a war! :boom:')
             ->embed(function (DiscordEmbed $embed) {
-                $embed->timestamp($this->notification->timestamp);
+                $embed->timestamp(carbon($this->notification->timestamp));
```

Comment:
what about casting the field with carbon in the model instead ?

Reply:
Can you do that? If so, it's probably a good idea